packages/styles/scss/components/multiselect/_multiselect.scss

```
@@ -42,6 +42,26 @@
     min-inline-size: auto;
   }
 
+  // When select all is turned on, the first item in the listbox
+  // is always the "Select all" option and gets special border
+  // treatment
+  .#{$prefix}--multi-select.#{$prefix}--multi-select--selectall
+    .#{$prefix}--list-box__menu-item:first-child
+    .#{$prefix}--list-box__menu-item__option {
+    padding: 0.6875rem 1rem;
```

```suggestion
    padding: 0.6875rem $spacing-05;
```

Should use token here. Also, what is the `0.6875` magic number? Should it be documented in the comments?